### PR TITLE
Use CMF 1.3  versions

### PR DESCRIPTION
**Pull request by WouterJ**
ping @lsmith77 @dbu

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
-        "doctrine/phpcr-bundle": "1.2.*",
-        "doctrine/phpcr-odm": "1.2.*",
-        "symfony-cmf/core-bundle": "1.2.*",
-        "symfony-cmf/content-bundle": "1.2.*",
+        "doctrine/phpcr-bundle": "1.3.*",
+        "doctrine/phpcr-odm": "1.3.*",
+        "symfony-cmf/core-bundle": "1.3.*",
+        "symfony-cmf/content-bundle": "1.3.*",
         "symfony-cmf/menu-bundle": "1.2.*",
         "symfony-cmf/block-bundle": "1.2.*",
```
Comment (WouterJ) [inline composer.json]: so this is really causing trouble:

* `symfony-cmf/block-bundle` 1.3 requires `sonata-project/block-bundle: 2.3.*`
* `sonata-project/admin-bundle` 2.3 requires `sonata-project/block-bundle: 2.2.*`
* `sonata-project/admin-bundle` 2.4-dev is not supported by the CMF

In other words: We can't use CmfBlockBundle 1.3 when using the SonataAdminBundle with the next CMF version.

**Comment by dbu** [comment]
ah, that will be confusing. but i assume there is no easy way to make block-bundle 1.3 work with sonata block bundle 2.2? 

is the block-bundle 1.2 incompatible with php 7 / symfony 2.7? if not i guess we can live with staying at the old version for one release. otherwise we need to either port back the support or move the changes that bump the sonata version to cmf block bundle 1.4 and remove them from the 1.3 version.

File: composer.json (hunk 1 above)
Comment (lsmith77) [inline composer.json]: and we cannot support 2.4 until Puli is stable .. right?

**Comment by lsmith77** [comment]
I guess there is no reason not to merge at least the above changes already .. right?

**Comment by WouterJ** [comment]
I believe so indeed

**Comment by WouterJ** [comment]
it'll probably fix https://travis-ci.org/symfony-cmf/standard-edition/jobs/82030290

**Pull request merged**